yb-voyager/src/srcdb/mysql.go

```diff
@@ -45,25 +45,18 @@ func (ms *MySQL) GetTableRowCount(tableName string) int64 {
 	return rowCount
 }
 
-func (ms *MySQL) GetTableApproxRowCount(tableProgressMetadata *utils.TableProgressMetadata) int64 {
+func (ms *MySQL) GetTableApproxRowCount(tableName *sqlname.SourceName) int64 {
 	var approxRowCount sql.NullInt64 // handles case: value of the row is null, default for int64 is 0
-	var query string
-	if !tableProgressMetadata.IsPartition {
-		query = fmt.Sprintf("SELECT table_rows from information_schema.tables "+
-			"where table_name = '%s'", tableProgressMetadata.TableName.ObjectName.Unquoted)
-	} else {
-		query = fmt.Sprintf("SELECT table_rows from information_schema.partitions "+
-			"where table_name='%s' and partition_name='%s' and table_schema='%s'",
-			tableProgressMetadata.ParentTable, tableProgressMetadata.TableName.ObjectName.Unquoted, tableProgressMetadata.TableName.SchemaName.Unquoted)
-	}
+	query := fmt.Sprintf("SELECT table_rows from information_schema.tables "+
+		"where table_name = '%s'", tableName.ObjectName.Unquoted)
```

Comment:
What if same table name is present in two different mysql databases?

Reply:
Yeah nice catch, fixed.